waku/v2/node/config.nim

```diff
@@ -55,12 +55,12 @@ type
 
     persistmessages* {.
       desc: "Enable message persistence: true|false",
-      defaultValue: false
+      defaultValue: true
```

Comment:
Do we want this to be true? To me the most common path is to be a light store client, but I guess one could argue both ways

Reply:
I'm leaning towards `false` as well. In my view, the most common "default" client is one that starts up, retrieves history as a "light" store node and participates in the network as a relay node. Thus, defaults of `store=true`, `relay=true`, `persist-messages=false`. Such clients will, of course, have to be configured with a `storenode` as well.